docs/3.api/2.components/4.nuxt-link.md

```diff
@@ -115,6 +115,7 @@ defineNuxtLink({
   activeClass?: string;
   exactActiveClass?: string;
   prefetchedClass?: string;
+  trailingSlashBehavior?: 'append' | 'remove'
```

Comment:
@manniL what do you think about making the option name `trailingSlash` - as we don't accept boolean (wise!), I think append/remove make clear what will happen.

Reply:
Yes, fine for me 👍